sources/procfs.go

```diff
@@ -620,9 +617,7 @@ func parseJobStatsText(jobStats string, promName string, helpText string) (metri
 		if err != nil {
 			return nil, err
 		}
-		for _, item := range jobList {
-			metricList = append(metricList, item)
-		}
+		metricList = append(metricList, jobList...)
```

Comment:
Do we want (need?) to check if `jobList` is not `nil` before this line? Say there aren't any jobs in the `job_stats` file (happens whenever no jobs have been running for a period of time), the for loop on [line 611](https://github.com/HewlettPackard/lustre_exporter/pull/85/files#diff-fde95e813ded08bf1be0acad8e83c4cfR611) would have zero iterations and `jobList` would never be initialized. I'm not sure how the `append(...)` command handles `nil` elements. If it handles them gracefully, no issue, otherwise we might want to add a check beforehand.